eventually-postgres/src/store.rs

```diff
@@ -86,12 +86,14 @@ const APPEND: &str = "SELECT * FROM append_to_store($1::text, $2::text, $3, $4,
 const CREATE_AGGREGATE_TYPE: &str = "SELECT * FROM create_aggregate_type($1::text)";
 
 const STREAM: &str = "SELECT e.*
-    FROM events e LEFT JOIN aggregates a ON a.id = e.aggregate_id
+    FROM events e LEFT JOIN aggregates a
+    ON a.id = e.aggregate_id AND a.aggregate_type_id = e.aggregate_type
```

Comment:
Minor: can you indent this?
```suggestion
        ON a.id = e.aggregate_id AND a.aggregate_type_id = e.aggregate_type
```